Aztec/Classes/TextKit/MediaAttachment.swift

```diff
@@ -15,6 +15,10 @@ protocol MediaAttachmentDelegate: class {
 ///
 open class MediaAttachment: NSTextAttachment
 {
+    /// Attributes accessible by the user, for general purposes.
+    ///
+    open var extraAttributes = [String: String]()
```

Comment:
Are we expecting this property to be overridden?

IMHO this should not be `open`.